src/plugins/data_views/server/register_index_pattern_usage_collection.ts

```diff
@@ -80,60 +85,74 @@ export async function getIndexPatternTelemetry(indexPatterns: DataViewsContract)
     },
   };
 
-  await ids.reduce(async (col, id) => {
-    await col;
-    const ip = await indexPatterns.get(id);
+  const findOptions: SavedObjectsCreatePointInTimeFinderOptions = {
+    type: DATA_VIEW_SAVED_OBJECT_TYPE,
+    perPage: 1000,
+    fields: ['attributes.fields', 'attributes.runtimeFieldMap'],
+  };
 
-    const scriptedFields = ip.getScriptedFields();
-    const runtimeFields = ip.fields.filter((fld) => !!fld.runtimeField);
+  const finder = savedObjectsService.createPointInTimeFinder<DataViewFieldAttrs>(findOptions);
```

Comment:
Should we also call `finder.close()` or it's not required?

Reply:
Its not needed with the for loop.